Lib/fontTools/ttLib/tables/_g_l_y_f.py

```diff
@@ -1165,6 +1235,8 @@ def getCoordinates(self, glyfTable):
                 allCoords.extend(coordinates)
                 allFlags.extend(flags)
             return allCoords, allEndPts, allFlags
+        elif self.isVarComposite():
+            raise NotImplementedError
```

Comment:
maybe add some error message to this, in case somebody attempts to use the `_g_l_y_f.Glyph.draw` and `drawPoints` method directly, suggesting to use the _TTGlyph.draw/drawPoints (which do handle VarComposites)